src/index.css

```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
 * {
    margin: 0;
    padding: 0;
```

Comment:
font Family ni bul jakka bersek tuurarak bolboibu?

Reply:
it's not working